---
layout: default
title: Metin Balaban | Research
---
<div class="container">
<h1>Research Projects</h1>
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive
    img-rounded" src="/images/apples.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Fast and scalable Distance-Based Phylogenetic Placement</h4>
  <p> 
Phylogenetic placement of query samples on an existing phylogeny is increasingly used in microbiome analyses and other biological studies. As the size of available reference trees used in microbiome analyses continues to grow, there is a growing need for methods that place sequences on ultra-large trees (e.g. 200,000 leaves)  with high accuracy. In this project, I developed an algorithm called APPLES (and its successor APPLES-2), a distance-based method for phylogenetic placement. In extensive studies, we showed that APPLES-2 is more accurate and scalable, even more so than some of the leading maximum likelihood methods. This scalability is owed to a divide-and-conquer technique that limits distance calculation and phylogenetic placement to parts of the tree most relevant to each query. APPLES-2 is available publicly at
  <a href="https://github.com/balabanmetin/apples"
  target="_blank">GitHub</a>.
  [Distance-based methods; genome skimming;phylogenetic placement.]
  </p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> Metin Balaban, Shahab Sarmashghi, Siavash Mirarab. <b>APPLES: Scalable
        Distance-Based Phylogenetic Placement with or without Alignments</b>.
      Systematic Biology, Volume 69, Issue 3, May 2020, Pages 566–578,
      <a href="https://doi.org/10.1093/sysbio/syz063"
       target="_blank">https://doi.org/10.1093/sysbio/syz063</a>.
    </li>
    <li>
	Metin Balaban, Yueyu Jiang, Daniel Roush, Qiyun Zhu, Siavash Mirarab. <b>Fast and Accurate Distance-based Phylogenetic Placement using Divide and Conquer.</b> Molecular Ecology Resources, 00, 1–15 (2021). <a href="https://doi. org/10.1111/1755-0998.13527"
       target="_blank">https://doi. org/10.1111/1755-0998.13527</a>.
    </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="300" height="300" class="img-responsive
    img-rounded" src="/images/misa.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Phylogenetic double placement of mixed samples</h4>
  <p> Consider a simple computational problem. The inputs are (i) the set of
  mixed reads generated from a sample that combines two organisms and (ii)
  separate sets of reads for several reference genomes of known origins. The
  goal is to find the two organisms that constitute the mixed sample. When
  constituents are absent from the reference set, we seek to phylogenetically
  position them with respect to the underlying tree of the reference species.
  In this project, we introduce a model based on Jaccard indices computed between each sample
  represented as k-mer sets. The model, built on several assumptions and
  approximations, allows us to formalize the phylogenetic double-placement
  problem as a non-convex optimization problem that decomposes mixture
  distances and performs phylogenetic placement simultaneously. Using a variety
  of techniques, we are able to solve this optimization problem numerically. 
  Resulting method called MISA and data are available at
  <a href="https://github.com/balabanmetin/misa"
      target="_blank">Github</a>.
  </p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> Metin Balaban,  Siavash Mirarab. <b>Phylogenetic double placement of
        mixed samples</b>. Bioinformatics 36, no. Supplement 1 (2020):i335–43. <a href="https://doi.org/10.1093/bioinformatics/btaa489"
       target="_blank">doi:10.1093/bioinformatics/btaa489</a>.
    </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="300" height="300" class="img-responsive
    img-rounded" src="/images/treecluster.jpg" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Clustering biological sequences using phylogenetic trees</h4>
  <p> Clustering homologous sequences based on their similarity is a problem that appears in many bioinformatics applications. The fact that sequences cluster is ultimately the result of their phylogenetic relationships. We defined a family of optimization problems that, given an arbitrary tree, return the minimum number of clusters such that all clusters adhere to constraints on their heterogeneity. These three problems can be solved in time that increases linearly with the size of the tree. We implement these algorithms in a tool called TreeCluster, which we test on three applications: OTU clustering for microbiome data, HIV transmission clustering, and divide-and-conquer multiple sequence alignment. We show that, by using tree-based distances, TreeCluster generates more internally consistent clusters than alternatives and improves the effectiveness of downstream applications. TreeCluster is available at
  <a href="https://github.com/niemasd/TreeCluster"
      target="_blank">Github</a>.
  </p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> Metin Balaban, Niema Moshiri, Uyen Mai, Xingfan Jia, Siavash Mirarab. <b>TreeCluster: Clustering biological sequences using phylogenetic trees</b>. PLOS ONE 14, no. 8 (2019): e0221068. 
      <a href="https://doi.org/10.1371/journal.pone.0221068"
       target="_blank">doi:10.1371/journal.pone.0221068</a>.
    </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="300" height="300" class="img-responsive
    img-rounded" src="/images/wol.png" />
  </div>
	Image credit to <a href="https://doi.org/10.1038/s41467-019-13443-4"
       target="_blank">Zhu et. al (2019)</a>.
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Automatic cataloguing of genomes of the multi-domain tree of life</h4>
  <p>  Much as we have maps to organize our geographical data, we need ways to organize the ever-increasing volume of genomic data. In RefSeq and other resources, more than half a million microbial genomes are now available and more are added everyday. 
Using trees for organizing all these genomes across domains of life is a principled and useful approach for two reasons:
1) trees are biologically meaningful representations, capturing the phylogenetic (i.e., evolutionary) relationships of species, albeit imperfectly, and 
2) they provide a convenient and scalable data structure that facilitates downstream analyses (e.g., retrieval and clustering). 
We are building a system for building and continuously updating of a tree-of-life of all available prokaryotic genomes where their phylogeny can be inferred and continuously updated, a wide range of downstream microbiome analyses will benefit. 
  </p>
  <h5>Ongoing Project</h5>
  <!--<ul class='pubs'>
    <li> Metin Balaban, Niema Moshiri, Uyen Mai, Xingfan Jia, Siavash Mirarab. <b>TreeCluster: Clustering biological sequences using phylogenetic trees</b>. PLOS ONE 14, no. 8 (2019): e0221068. 
      <a href="https://doi.org/10.1371/journal.pone.0221068"
       target="_blank">doi:10.1371/journal.pone.0221068</a>.
    </li>
  </ul>
-->
</div>


<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="500" height="300" class="img-responsive
    img-rounded" src="/images/tk4.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Genome-wide alignment-free distance estimation under a no strand-bias model</h4>
  <p> 
The alignment-free methods have much appeal in terms of simplifying the process of inference, especially when analyzing genome-wide data. 
Despite the appeal, one limitation of alignment-free methods is that they typically rely on simplified models of sequence evolution such as Jukes-Cantor.
It is possible to compute pairwise distances under more complex models by computing frequencies of base substitutions provided that these quantities can be estimated in the alignment-free setting. An underappreciated limitation is that for many forms of genome-wide data, which arguably present the best use case for alignment-free methods, the strand of DNA sequences is unknown. 
Under such conditions, the no-strand bias models are the best that we can do. Here, we show how to calculate distances under a no-strain bias restriction of the General Time Reversible (GTR) model called TK4 without relying on alignments or even assemblies using k-mers. 
 
  </p>
  <h5>Ongoing Project</h5>
  <!--<ul class='pubs'>
    <li> Metin Balaban, Niema Moshiri, Uyen Mai, Xingfan Jia, Siavash Mirarab. <b>TreeCluster: Clustering biological sequences using phylogenetic trees</b>. PLOS ONE 14, no. 8 (2019): e0221068. 
      <a href="https://doi.org/10.1371/journal.pone.0221068"
       target="_blank">doi:10.1371/journal.pone.0221068</a>.
    </li>
  </ul>
-->
</div>
<!--
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/aquarium_ecoli.png" />
  </div>
</div>

<div class="col-md-8" style='margin-top:20px'>
  <h4>The Aquarium cloning workflows</h4>
  <p><a href="http://klavinslab.org/aquarium.html" target="_blank">Aquarium</a> is a smart laboratory operating system invented in the <a href="http://klavinslab.org" target="_blank">Klavins Lab</a> where researchers can specify how to manipulate items that stored in the inventory using protocols and workflows written in computer codes. Adapted from the standard molecular cloning workflows, I wrote a majority of the codes in Ruby implementing two core molecular cloning workflows: plasmid construction and yeast cloning. These workflows take and batch users' inputs to the workflow and semi-automatically schedule the entire cloning process from ordering primers to Sanger sequencing verifications, resulting a nearly effortless cloning process in Aquarium. This technology also led to the startup <a href="http://www.aquarium.bio/" target="_blank">Aquarium</a> where I work as an engineer to build out the 21st century laboratory operating system, enabling reproducible and transferable workflows in biology labs and more.<p>
  <h5>Related news</h5>
  <ul class='pubs'>
  <li> Aquarium Lab in the Cloud. <b>Engineering research projects haul in CoMotion Innovation Fund dollars</b>. News, University of Washington, 2015. <a href='https://www.engr.washington.edu/news/comotion-innovation-fund-2015' target="_blank"><span class="glyphicon glyphicon-link"></span></a>
  </li>
  </ul>
</div>
<!--
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/frequency.png" />
  </div>
</div>

<div class="col-md-8" style='margin-top:20px'>
  <h4>Frequency response for a synthetic auxin signaling pathway in yeast</h4>
  <p>I developed an experimental process in the <a href="http://klavinslab.org" target="_blank">Klavins Lab</a> to interrogate frequency response for a synthetic auxin signaling pathway in yeast. The approach was using a microfluidic device called <a href="http://www.emdmillipore.com/US/en/life-science-research/cell-culture-systems/cellASIC-live-cell-analysis/microfluidic-plates/68eb.qB.wfkAAAFBWmVb3.sJ,nav" target="_blank">CellASIC</a> to grow the yeast cell, where one can control the system precisely with different frequencies of auxin inputs and observe using fluorescence microscopy over time. I wrote Matlab based software to do image processing on the time course fluorescence images, which resulted in quantified time-course fluorescence data on the cells. Using the data, I generated a <a href="http://en.wikipedia.org/wiki/Bode_plot" target="_blank">Bode plot</a> describing the frequency response of signaling pathway. The result showed that this pathway works as a low-pass filter.</p>
  <h5>Related work</h5>
  <ul class='pubs'>
  <li> Y. Yang, E. Klavins. <b>Frequency domain signal processing in biological system</b>. Poster and Lightning	Talk, <a href='http://molecular-programming.org/workshops/one.html' target="_blank"><i>Molecular Programming Project Workshop</i></a>, Oxnard, CA, USA, 2013.
  </li>
  </ul>
</div>
-->
