---
layout: default
title: Metin Balaban | Research
---
<div class="container">
<h1>Research Projects</h1>
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive
    img-rounded" src="/images/apples.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>APPLES: Scalable Distance-Based Phylogenetic Placement with or without Alignments</h4>
  <p> In this project, I developed an algorithm called APPLES, 
  a distance-based method for phylogenetic placement. Compared to ML, APPLES is
  an order of magnitude faster and more memory efficient, and unlike ML, it is
  able to place on large backbone trees (tested for up to 200,000 leaves). We
  show that using dense references improves accuracy substantially so that
  APPLES on dense trees is more accurate than ML on sparser trees, where it can
  run. Finally, APPLES can accurately identify samples without assembled
  reference or aligned queries using kmer-based distances, a scenario that ML
  cannot handle. APPLES is available publically at
  <a href="https://github.com/balabanmetin/apples"
  target="_blank">Github</a>.
  [Distance-based methods; genome skimming;phylogenetic placement.]
  </p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> Metin Balaban, Shahab Sarmashghi, Siavash Mirarab. <b>APPLES: Scalable
        Distance-Based Phylogenetic Placement with or without Alignments</b>.
      Systematic Biology, Volume 69, Issue 3, May 2020, Pages 566â€“578,
      <a href="https://doi.org/10.1093/sysbio/syz063"
       target="_blank">https://doi.org/10.1093/sysbio/syz063</a>.
    </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive
    img-rounded" src="/images/apples.png" />
  </div>
</div>
<div class="col-md-8" style='margin-top:20px'>
  <h4>Phylogenetic double placement of mixed samples</h4>
  <p> Consider a simple computational problem. The inputs are (i) the set of
  mixed reads generated from a sample that combines two organisms and (ii)
  separate sets of reads for several reference genomes of known origins. The
  goal is to find the two organisms that constitute the mixed sample. When
  constituents are absent from the reference set, we seek to phylogenetically
  position them with respect to the underlying tree of the reference species.
  In this project, we introduce a model that relates distances between a mixed sample and
  reference species to the distances between constituents and reference
  species. Our model is based on Jaccard indices computed between each sample
  represented as k-mer sets. The model, built on several assumptions and
  approximations, allows us to formalize the phylogenetic double-placement
  problem as a non- convex optimization problem that decomposes mixture
  distances and performs phylogenetic placement simultaneously. Using a variety
  of techniques, we are able to solve this optimization problem numerically. We
  test the resulting method, called MISA, on a varied set of simulated and
  biological datasets. Despite all the assumptions used, the method performs
  remarkably well in practice. The sofware and data are available at
  <a href="https://github.com/balabanmetin/misa"
      target="_blank">Github</a>.
  </p>
  <h5>Related work</h5>
  <ul class='pubs'>
    <li> Metin Balaban,  Siavash Mirarab. <b>Phylogenetic double placement of
        mixed samples</b>.
      Bioinformatics, in press.
    </li>
  </ul>
</div>


<!--
<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/aquarium_ecoli.png" />
  </div>
</div>

<div class="col-md-8" style='margin-top:20px'>
  <h4>The Aquarium cloning workflows</h4>
  <p><a href="http://klavinslab.org/aquarium.html" target="_blank">Aquarium</a> is a smart laboratory operating system invented in the <a href="http://klavinslab.org" target="_blank">Klavins Lab</a> where researchers can specify how to manipulate items that stored in the inventory using protocols and workflows written in computer codes. Adapted from the standard molecular cloning workflows, I wrote a majority of the codes in Ruby implementing two core molecular cloning workflows: plasmid construction and yeast cloning. These workflows take and batch users' inputs to the workflow and semi-automatically schedule the entire cloning process from ordering primers to Sanger sequencing verifications, resulting a nearly effortless cloning process in Aquarium. This technology also led to the startup <a href="http://www.aquarium.bio/" target="_blank">Aquarium</a> where I work as an engineer to build out the 21st century laboratory operating system, enabling reproducible and transferable workflows in biology labs and more.<p>
  <h5>Related news</h5>
  <ul class='pubs'>
  <li> Aquarium Lab in the Cloud. <b>Engineering research projects haul in CoMotion Innovation Fund dollars</b>. News, University of Washington, 2015. <a href='https://www.engr.washington.edu/news/comotion-innovation-fund-2015' target="_blank"><span class="glyphicon glyphicon-link"></span></a>
  </li>
  </ul>
</div>

<div class="col-md-4" style='margin-top:20px'>
  <div class="imgAbt">
    <img style = 'margin: auto' width="260" height="260" class="img-responsive img-rounded" src="/images/frequency.png" />
  </div>
</div>

<div class="col-md-8" style='margin-top:20px'>
  <h4>Frequency response for a synthetic auxin signaling pathway in yeast</h4>
  <p>I developed an experimental process in the <a href="http://klavinslab.org" target="_blank">Klavins Lab</a> to interrogate frequency response for a synthetic auxin signaling pathway in yeast. The approach was using a microfluidic device called <a href="http://www.emdmillipore.com/US/en/life-science-research/cell-culture-systems/cellASIC-live-cell-analysis/microfluidic-plates/68eb.qB.wfkAAAFBWmVb3.sJ,nav" target="_blank">CellASIC</a> to grow the yeast cell, where one can control the system precisely with different frequencies of auxin inputs and observe using fluorescence microscopy over time. I wrote Matlab based software to do image processing on the time course fluorescence images, which resulted in quantified time-course fluorescence data on the cells. Using the data, I generated a <a href="http://en.wikipedia.org/wiki/Bode_plot" target="_blank">Bode plot</a> describing the frequency response of signaling pathway. The result showed that this pathway works as a low-pass filter.</p>
  <h5>Related work</h5>
  <ul class='pubs'>
  <li> Y. Yang, E. Klavins. <b>Frequency domain signal processing in biological system</b>. Poster and Lightning	Talk, <a href='http://molecular-programming.org/workshops/one.html' target="_blank"><i>Molecular Programming Project Workshop</i></a>, Oxnard, CA, USA, 2013.
  </li>
  </ul>
</div>
-->
